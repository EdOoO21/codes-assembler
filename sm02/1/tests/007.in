941 -692 -192 333 -902 -852 681 97 -808 -252 193 -882 863 39 -561 -924 -824 -112 -144 -857 -508 -815 128 -131 -879 693 158 -747 940 -543 291 284 193 940 -874 181 199 -188 -899 999 -548
