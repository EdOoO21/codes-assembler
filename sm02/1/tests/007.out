-548
999
-899
-188
199
181
-874
940
193
284
291
-543
940
-747
158
693
-879
-131
128
-815
-508
-857
-144
-112
-824
-924
-561
39
863
-882
193
-252
-808
97
681
-852
-902
333
-192
-692
941
